src/prelude.rs

```diff
@@ -3,5 +3,5 @@
 pub use app::{Application, ApplicationBuilder};
 pub use config::Config;
 pub use engine::Engine;
-pub use renderer::input::*;
+//pub use renderer::input::*;
```

Comment:
I'm not yet sure what we should do with this prelude module, let's decide once [rs] PRs are merged

Reply:
See above